packages/flutter/lib/src/material/radio.dart

```diff
@@ -75,22 +367,17 @@ const double _kInnerRadius = 4.5;
 class Radio<T> extends StatefulWidget {
   /// Creates a Material Design radio button.
   ///
-  /// The radio button itself does not maintain any state. Instead, when the
-  /// radio button is selected, the widget calls the [onChanged] callback. Most
-  /// widgets that use a radio button will listen for the [onChanged] callback
-  /// and rebuild the radio button with a new [groupValue] to update the visual
-  /// appearance of the radio button.
+  /// {@macro flutter.material.Radio.groupValue}
   ///
   /// The following arguments are required:
   ///
-  /// * [value] and [groupValue] together determine whether the radio button is
-  ///   selected.
+  /// * [value] is used to identify this radio
   /// * [onChanged] is called when the user selects this radio button.
   const Radio({
     super.key,
     required this.value,
-    required this.groupValue,
-    required this.onChanged,
+    this.groupValue,
```

Comment:
Should we deprecate `groupValue` and really encourage folks to migrate to RadioGroup so that folks are using best practices for accessibility?
The fact that this also reduces verbosity is a huge win.